### PR TITLE
Fix for crash while adding custom host

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKNewLoginHostViewController.m [hunk 1]
```diff
@@ -78,7 +78,6 @@ - (void)viewWillAppear:(BOOL)animated {
  */
 - (void)addNewServer:(id)sender {
     [self.loginHostListViewController addLoginHost:[SFSDKLoginHost hostWithName:self.name.text host:self.server.text deletable:YES]];
-    [self.navigationController popViewControllerAnimated:YES];
```
Comment (bhariharan) [inline libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/SFSDKNewLoginHostViewController.m]: The view controller was being popped in 2 different places.

**Comment by wmathurin** [comment]
:+1:

**Comment by kchitalia** [comment]
Just curious, at what point was it crashing? I was able to successfully add
gus.my.salesforce.com and login without this fix.

Sent from my iPhone

On Feb 22, 2016, at 5:17 PM, Bharath Hariharan <notifications@github.com>
wrote:

Merged #1411
<https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1411>.

—
Reply to this email directly or view it on GitHub
<https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1411#event-560749263>
.

**Comment by bhariharan** [comment]
@kchitalia Start up any sample app -> Click gear icon -> Click '+' button to add new custom host -> Type in valid Name and URL -> Click 'Done' button -> Crash. I tested on ```iOS 8.4``` and ```iOS 9.1``` emulators. How did you add ```GUS```?

**Comment by kchitalia** [comment]
That's interesting. I used the same steps to add gus.my.salesforce.com

Sent from my iPhone

On Feb 22, 2016, at 6:29 PM, Bharath Hariharan <notifications@github.com>
wrote:

@kchitalia <https://github.com/kchitalia> Start up any sample app -> Click
gear icon -> Click '+' button to add new custom host -> Type in valid Name
and URL -> Click 'Done' button -> Crash. I tested on iOS 8.4 and iOS 9.1
emulators. How did you add GUS?

—
Reply to this email directly or view it on GitHub
<https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1411#issuecomment-187489337>
.

**Pull request merged**